epicfocus/README.md

```diff
@@ -0,0 +1,129 @@
+
+There are [many reasons](https://www.forbes.com/sites/carolkinseygoman/2014/03/02/the-story-of-why-people-dont-collaborate/) why people don't collaborate. They don't see the benefits of it, are afraid of being wrong and therefore vulnerable, don't trust their teammates, or they've tried before only to have their opinion ignored. Or maybe there is nothing to collaborate on?
+
+### Team has nothing to present
+As a coach, I like to use the sprint review as a vantage point to do observations. Do the sprint review meetings happen regularly? Do the teams have something to review? Do you feel like there is continuous progress? Perhaps you catch yourself wondering what the teams have been doing the entire sprint as they have very little or nothing to review.
+
+> "If you can't measure it, you can't improve it."
+> - Peter Drucker
+
+## Measuring the process with the Value Stream Map
+The symptoms we observe are a good starting point for change. We now want to support these observations with data. One way to examine your development process is to [create a value stream map or VSM](https://xebia.com/blog/how-to-create-a-value-stream-map/). It is an inventory of all the activities that happen before a piece of software hits production. You also want to know how much time each activity takes. Make sure to record all the waiting times, such as waiting for code review, waiting for approval(s), waiting for deploy, etc. Once you have that, you can put them in two buckets: value-adding and non-value adding activities or, in other words, waste. It is quite intuitive that activities such as writing code, writing tests, and reviewing the code, are considered as value-adding while all kinds of waits belong to waste.
+
+Here is the visualization of our VSM and how it changed over time as we tweaked the process.
+![VSM](images/vsm.png?raw=true "Value Stream Map")
+
+By performing this exercise, you will identify the waste in your process, and it will become clear what parts of it need change. Of course, you still don't know what work items are valuable by themselves. For that, you need another metric.
+
+## Drilling down into the backlog with the Epic Focus Metrics
+#### 1. Identify value-adding and value-retaining backlog items
+By building a value stream map and eliminating waste, you can speed up your processes. We would like to offer you a similar exercise only performed not against activities, but against your backlog items. This exercise can give you insight into the speed of your feature delivery and your focus on value.
+
+Put all the completed work into two buckets: value-adding and value-retaining. We chose to mark new features, or epics, as value-adding and bugs and technical debt items as value-retaining. Both are important; we need to add new value while maintaining the existing value of our products.
+
+![Distribution of tickets](images/adding-vs-retaining.png?raw=true "Distribution of work items")
```

Comment:
I think this image contains too much text, which makes it slightly harder to comprehend. You could put the percentage values inside the pie slices, and remove those "pointer" lines.